ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/BigSampler.scala

```diff
@@ -122,6 +122,7 @@ object BigSampler extends Command {
         |  [--distribution=(uniform|stratified)]      An optional arg to sample for a stratified or uniform distribution. Must provide `distributionFields`
         |  [--distributionFields=<field1,field2,...>] An optional list of fields to sample for distribution. Must provide `distribution`
         |  [--exact]                                  An optional arg for higher precision distribution sampling.
+        |  [--byteHasher=(bytes|hex|base64)]          An optional arg for how to hash fields of type bytes: raw bytes, hex encoded string, or base64 encoded string. Default is to hash raw bytes.
```

Comment:
Actually another nitpick, maybe we should just have it be optional to provide hex or base64 and use bytes otherwise if unspecified. Sorry for the back and forth

Reply:
What happens if I have multiple bytes fields? This modification is applied across all of them?

Reply:
It is optional and the default is bytes. The modification will be applied across all fields.